utilities/xmlrpcpp/test/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
-catkin_add_gtest(xmlrpcvalue_base64 xmlrpcvalue_base64.cpp)
-target_link_libraries(xmlrpcvalue_base64 xmlrpcpp)
+catkin_add_gtest(${PROJECT_NAME}-xmlrpcvalue_base64 xmlrpcvalue_base64.cpp)
```

Comment:
There is no need to rename the target. Please keep the target babe as is and only wrap the `target_link_libraries` in the conditional block.